### PR TITLE
Bug 1220814 - Imagecompare: test_settings_NC_RTL.py fails while waiting for call barring dialog

### PR DESCRIPTION
**Pull request by nojunpark**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3cb50390de4ea8bd11952351c640c3ccf55e41dc

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a0bfdcc8aebcda574f223f46aa2bed6129c37059

File: tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/call_settings.py [hunk 1]
```diff
@@ -239,8 +239,8 @@ def tap_call_barring(self):
         Wait(self.marionette).until(expected.element_displayed(*self._call_settings_query_locator))
 
     def wait_until_call_barring_info_received(self):
-        Wait(self.marionette, timeout=60).until(
-            expected.element_displayed(*self._call_barring_status_disabled_text_locator))
+        Wait(self.marionette, timeout=120, interval=1).until(
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/call_settings.py]: Wow, can this really take up to 2 minutes?

Comment (npark-mozilla) [inline tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/call_settings.py]: not on local device, but i learned that on jenkins it takes much slower.  looks like it's taking around 60 seconds, so I doubled it.  The script that takes less than 2 min locally, it takes 6 minutes on jenkins according to the timestamp, for some reason.

**Pull request merged**